server/cmwell-bg/src/main/scala/cmwell/bg/CMWellBGActor.scala

```diff
@@ -84,6 +89,20 @@ class CMWellBGActor(partition: Int,
     logger.info(s"CMwellBGActor-$partition starting")
     super.preStart()
     self ! Start
+
+    import akka.pattern.ask
+    sys.addShutdownHook {
```

Comment:
consider this scenario:
on a particular JVM:
1. BGActor5 is run on it.
2. sys.addShutdownHook {BActor5 ! shutDown}
3. BGActor5 crashes - no one runs the shutdown???
4. BGActor5 starts on another JVM
5. the original JVM crashes - BGActor5 is shutdown!!!

Reply:
Do you see any reason why no to add the shutdown logic to the postStop of the BGActor?

Reply:
`postStop` sounds like a better plan. Will that answer the scenario you described?